StuDash_home/models.py

```diff
@@ -1,3 +1,25 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import User
+# course= models.ForeignKey(Course) will be added to this class
+# after class Course will be created by other team member
+# descripition- Users are related to certain courses.
+# for each course, the user is able to create relevant bookmarks
+# each bookmark stores url, name and last visit date in order to enable potential future filtering
 
-# Create your models here.
+
+class Bookmark (models.Model):
+    user = models.ManyToManyField(User, blank=True)
+    url = models.URLField(max_length=200, primary_key=True)
```

Comment:
Why do you want that field to be a primary key?
The table usually has an ID column (I think default by Django generates it), which is the primary key.